howso/details.amlg

```diff
@@ -426,9 +435,35 @@
 			(call !ComputeCaseMDA)
 		)
 
+		(if (and (!= 0 (size (get details "prediction_stats"))) (not (get details "get_prediction_stats")))
+			(accum (assoc
+				warnings 
+					(associate (concat 
+						"'get_prediction_stats' is set to false. The stats selected in 'prediction_stats' "
+						"will not be returned unless 'get_prediction_stats' is true."
+					))
+			))
+		)
+
+		;If the confusion_matrix is selected, then all of the stats are returned.
+		(if (get details "confusion_matrix")
+			(accum (assoc details (assoc "prediction_stats" (true))))
+		)
+
 		;append feature errors and deviations
-		(if (get details "feature_residuals")
-			(call !ComputeReactFeatureResiduals)
+		(if (or (get details "prediction_stats") (get details "feature_residuals"))
+			(seq
+				(if (get details "prediction_stats")
+					(assign (assoc 
```

Comment:
are `compute_all_statistics` and `store_values` declared earlier?

We should verify this `(assign)` is working as expected... I'd hate for these local stats to get cached in the global map.